rust/worker/src/blockstore/arrow/blockfile.rs

```diff
@@ -275,6 +276,30 @@ impl<'me, K: ArrowReadableKey<'me> + Into<KeyWrapper>, V: ArrowReadableValue<'me
         None
     }
 
+    pub(super) async fn load_blocks(&self, block_ids: Vec<Uuid>) -> () {
+        let mut futures = Vec::new();
+        for block_id in block_ids {
+            futures.push(self.get_block(block_id));
+        }
+        join_all(futures).await;
+    }
+
+    pub(crate) async fn load_blocks_for_keys(&self, prefixes: &Vec<&str>, keys: &Vec<K>) -> () {
```

Comment:
We should not do &Vec<> prefer slice types

Reply:
Removed the borrow altogether